### ISSUE TITLE
Many error symbols not included

### ISSUE THREAD

**Issue by njbotkin** [sidorares/node-mysql2#1924]
We noticed that a 3140: [ER_INVALID_JSON_TEXT](https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html#error_er_invalid_json_text) isn't included in https://github.com/sidorares/node-mysql2/blob/master/lib/constants/errors.js.  Then we noticed a lot more error symbols aren't included.  Is there a reason for this?  Would you like a PR that includes the rest of the codes and symbols in https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html?

**Comment by sidorares** [comment]
> Would you like a PR that includes the rest of the codes and symbols in https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html?

Yes please. No specific reason, just not in sync with newer error codes added recentlyю First commit to errors.js is from 2013.

**Comment by elimelt** [comment]
Is anyone currently working on this issue? I'd be happy to help!

**Comment by sidorares** [comment]
@elimelt nothing in progress afaik, happy to review your PR

**Comment by elimelt** [comment]
I've thus far been able to update generate-error-constants.js (found in node mysql module) to work with the source code for MySQL 8.0, but a lot of the codes have either been marked as OBSOLETE, or have been changed entirely. It seems like it was already the approach to just remove the OBSOLETE tags on the error codes, so I went ahead and did that, but I am a little more reluctant to push changes that overwrite/change codes entirely. 

Although, since errors.js basically just serves as a lookup table, I doubt this will introduce any compatability issues accross the codebase. For now I will just include the most up-to-date error codes from the MySQL 8.0 source, and if the maintainers want to go a different direction then I'm happy to find an alternative solution upon review of my PR. Perhaps this would be maintaining a different verion of error.js depending on the version of MySQL one is using?
